commcare-cloud/commcare_cloud/environment/main.py

```diff
@@ -37,10 +38,36 @@ def check(self):
         self.proxy_config
         self.create_generated_yml()
 
+    @property
+    def env_name(self):
```

Comment:
I'm almost tempted to call this `get_env_name_for_logging` so no one is ever tricked into using this value instead of `deploy_env` for anything other than that. I'm not going to be militant about it, but do think that would reduce future errors.

Reply:
Good point, I've switched to using `deploy_env` and removed this altogether: 310b5a4